### ISSUE TITLE
The mIoU on the cityscapes dataset is low

### ISSUE THREAD

**Issue by fyn111** [NVlabs/SegFormer#53]
I use local_configs/segformer/B5/segformer.b5.1024x1024.city.160k.py to train a model, but the mIoU is low even after 63950 iterations:
+---------------+-------+-------+
| Class         | IoU   | Acc   |
+---------------+-------+-------+
| road          | 0.81  | 39.61 |
| sidewalk      | 90.83 | 96.99 |
| building      | 68.91 | 94.91 |
| wall          | 22.03 | 24.51 |
| fence         | 13.55 | 51.07 |
| pole          | 11.99 | 56.55 |
| traffic light | 11.77 | 20.65 |
| traffic sign  | 89.88 | 97.59 |
| vegetation    | 55.71 | 82.64 |
| terrain       | 21.43 | 85.35 |
| sky           | 8.41  | 9.78  |
| person        | 57.04 | 97.61 |
| rider         | 27.36 | 68.83 |
| car           | 22.67 | 77.23 |
| truck         | 19.68 | 47.09 |
| bus           | 13.06 | 65.01 |
| train         | 0.0   | nan   |
| motorcycle    | 31.27 | 75.63 |
| bicycle       | 0.46  | 0.47  |
+---------------+-------+-------+
2021-09-20 08:44:46,087 - mmseg - INFO - Summary:
2021-09-20 08:44:46,088 - mmseg - INFO - 
+--------+-------+-------+-------+
| Scope  | mIoU  | mAcc  | aAcc  |
+--------+-------+-------+-------+
| global | 29.83 | 60.64 | 90.59 |
+--------+-------+-------+-------+

**Comment by Inch-Z** [comment]
I have the same question on the ADE20K, about 0.6% lower than the paper report

**Comment by Mordokkai** [comment]
I also have the same problem on ADE20K. I downloaded ADE20K dataset, downloaded the pretrained weights mit_b5.pth. Then I run `./tools/dist_train.sh local_configs/segformer/B5/segformer.b5.640x640.ade.160k.py 1` and get a very bad result of 27.87 mIoU compared to your 51 mIoU in your paper.
You can check my full log here: [log.txt](https://github.com/NVlabs/SegFormer/files/7379547/log.txt)

**Comment by prismleong** [comment]
The paper says they pre-trained the model on Image-net, have you tried pre-training?

**Comment by cuiziteng** [comment]
It seem you haven't load pre-train model or just train on 1 GPUs and didn't adjust learning rate. Here is my results:

+---------------+-------+-------+
|     Class     |  IoU  |  Acc  |
+---------------+-------+-------+
|      road     | 98.33 | 99.02 |
|    sidewalk   | 86.38 | 93.75 |
|    building   | 93.55 | 97.01 |
|      wall     |  67.6 | 74.32 |
|     fence     | 64.07 | 71.14 |
|      pole     | 69.53 | 79.94 |
| traffic light | 75.17 | 84.61 |
|  traffic sign | 81.84 | 88.69 |
|   vegetation  | 93.12 | 96.99 |
|    terrain    | 66.42 |  74.3 |
|      sky      |  95.3 | 98.57 |
|     person    | 84.98 | 92.43 |
|     rider     | 68.77 | 82.45 |
|      car      | 95.89 | 98.07 |
|     truck     | 87.63 |  90.3 |
|      bus      | 90.39 | 95.18 |
|     train     | 82.65 | 87.15 |
|   motorcycle  |  73.5 |  83.6 |
|    bicycle    | 80.45 | 90.49 |
+---------------+-------+-------+
2022-04-09 14:31:53,589 - mmseg - INFO - Summary:
2022-04-09 14:31:53,589 - mmseg - INFO - 
+-------+-------+-------+
|  aAcc |  mIoU |  mAcc |
+-------+-------+-------+
| 96.62 | 81.87 | 88.32 |
+-------+-------+-------+

**Comment by xibi777** [comment]
I train b0_cityscapes on 1 GPU, and I only change the samples_per_gpu into 8. It performs not well, and it seems that the model overfit after 60000 iterations. Should I also change my learning rate? Thanks!!!
